docs/console-api/single-page.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Single Page
+sidebar_label: Single Page
```

Comment:
I'd call this: All Endpoints

Reply:
[f087697](https://github.com/statsig-io/docs/pull/294/commits/f08769708521e7857b3d8f90ea0b93eebac95c9d)